bindings/python/python/shapleyrs/__init__.py

```
@@ -0,0 +1,8 @@
+from ._shapley import *
```

Ruff (0.11.9)

`from ._shapley import *` used; unable to detect undefined names

(F403)